app/org/sagebionetworks/bridge/upload/IosSchemaValidationHandler2.java

```diff
@@ -332,11 +333,14 @@ private static void validateInfoJsonFileList(UploadValidationContext context, St
         }
 
         if (createdOn == null) {
-            // Recover by using current time.
+            // Recover by using current time. Don't set a timezone, since it's indeterminate.
             context.addMessage(String.format("upload ID %s has no timestamps, using current time", uploadId));
-            createdOn = DateUtils.getCurrentDateTime();
+            recordBuilder.withCreatedOn(DateUtils.getCurrentMillisFromEpoch());
```

Comment:
Discussed offline. If the users don't submit a timezone, instead of submitting a default timezone, we should submit no timezone.